### DISCUSSION TITLE
forge modeline status disappears after quitting magit status buffer

### DISCUSSION THREAD

**Discussion by LemonBreezes** [magit/forge#633] [Ideas]
Basically, the problem is that when I use `q` to exit `magit-status` and then `C-x g` again, the `mode-line-process` from `forge--msg` is lost. Can we make it so that `forge--msg` has some indirection and instead sets the status for a general Git repository and then have separate hooks that sync that `mode-line-process` value to the `magit-status` buffers? That way, the status doesn't get lost during a long `forge-pull` even if I kill the `magit-status` buffer and reopen it.

**Comment by tarsius** [comment]
(Please use "discussions" to suggest new features or improvements to existing features.)

I am currently dealing with much larger changes[^1], and will delay any decision about this request, until I am ready to return to and focus on such details.

[^1]: Such as adding support for "discussions" to Forge.

**Comment by tarsius** [comment]
I went with a different approach; optionally also showing the progress in the echo area.

This is controlled by the new variable `ghub-graphql-message-progress`. See also #744.

I do not intend to implement what was suggested here.
